mempool/src/core_mempool/transaction.rs

```diff
@@ -111,6 +111,8 @@ pub enum SubmittedBy {
 #[derive(Debug, Clone)]
 pub struct InsertionInfo {
     pub insertion_time: SystemTime,
+    pub ready_time: SystemTime,
```

Comment:
I am a bit confused:

if part_time is not None, then it is same as insertion_time, correct?
I understand tracking ready_time, but we are not using it? shouldn't we be looking at SystemTime::now().duration_since(ready_time), instead of taking insertion time and then subtracting parked_duration ?

also parked_duration should be looking from parked to ready, not from insertion to parked?

Reply:
after that is fine, but it might be simpler to do:


`let commit_minus_parked = SystemTime::now().duration_since(insertion_info.park_time.unwrap_or(insertion_info.insertion_time))`

instead of doing actual minus?